src/services/logger.ts

```diff
@@ -0,0 +1,22 @@
+import { Logger } from '@aws-lambda-powertools/logger'
+import { LogLevel } from '@aws-lambda-powertools/logger/lib/types/Log'
+import { Context } from 'aws-lambda'
+
+const getLogLevel = (): LogLevel => {
+  if (process.env.LOG_LEVEL) {
+    return process.env.LOG_LEVEL as LogLevel
+  }
+
+  return 'DEBUG'
+}
```

Comment:
As a point of interest (maybe), this fn _could_ be expressed:
`const getLogLevel = (): LogLevel => process.env.LOG_LEVEL || 'DEBUG';`
which is functionally the same.

Reply:
It doesn't seem to be, there's a typescript thing that pops up. Try it!

Reply:
ah ok - not sure why as `LogLevel` type incorporates `string` and I don't see that error my end. Adding `as LogLevel` for the env var as you did before should fix that.